_includes/footer.html

```diff
@@ -13,6 +13,16 @@
     <span><a href="/the-team/">The team</a></span>
     <span><a href="/explainers/">Explainers</a></span>
   </div>
+  <div class="footer-info">
+    <span>
+      <img src="/img/social-09.svg" alt="linkedin logo" />
```

Comment:
Just realized: accessibility: these images should have `alt=""` to indicate that what is communicated by the image is also communicated by the text next to it.

Also, alt texts should never say "image" or "logo" or similar, as that is implied/already read out loud by the screen readers when it is an `img` tag.